lib/bibliothecary/runner.rb

```diff
@@ -61,6 +61,10 @@ def load_file_info_list(path)
       file_list
     end
 
+    def find_manifests(path)
+      load_file_info_list(path).reject { |info| info.package_manager.nil? }.map { |info| info.relative_path }
+    end
```

Comment:
If you make this take an optional ignore_unparseable_files boolean, then the `analyse` method above could call this one and avoid a little cut-and-paste I think